configuration/src/main/java/io/airlift/configuration/ConfigurationAwareModule.java

```diff
@@ -16,11 +16,41 @@
 package io.airlift.configuration;
 
 import com.google.common.annotations.Beta;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
 import com.google.inject.Module;
 
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
 @Beta
 public interface ConfigurationAwareModule
         extends Module
 {
     void setConfigurationFactory(ConfigurationFactory configurationFactory);
+
+    static Module combine(Module... modules)
```

Comment:
I don't like interfaces poluted with utility static methods? What is the motivation behind this?

Reply:
Your thinking is old-fashioned. Static methods were added to interfaces so that separate utility classes are not required. If you look at all the new JDK APIs, the static utility methods go on the interfaces.